app-laptop/thinkfan/thinkfan-1.0.1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils systemd
+
+DESCRIPTION="A simple and lightweight fan control program"
+HOMEPAGE="https://github.com/vmatare/thinkfan/"
+SRC_URI="https://github.com/vmatare/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="atasmart nvml +yaml"
+
+DEPEND="
+	atasmart? ( dev-libs/libatasmart:= )
+	yaml? ( dev-cpp/yaml-cpp:= )"
+
+RDEPEND="
+	${RDEPEND}
+	nvml? ( x11-drivers/nvidia-drivers )
+"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	# Don't install license file
+	# Don't use supplied init scripts
+	# Fix QA warning for doc dir
+	local mysedargs=(
```

Comment:
What's the point of this? You can just use sed here and shorten the ebuild for 3 lines. In my opinion, this makes it less obvious what's happening in here.

Reply:
I think, this gives a better readability, instead of having a long single line.

Reply:
I mean, 
```
sed -i -e 's/COPYING//' \
	-e 's/OPENRC_FOUND/DISABLE/g' \
	-e 's/SYSTEMD_FOUND/DISABLE/g' \
	-e "s/thinkfan)/${PF})/" \
	-CMakeLists.txt || die
```

is 4 lines shorter (3 if you drop first -e to its own line), doesnt add unnecessary complexity, and is more self-explanatory than your magic in here ;)

also looks like Zlogene already took this package, remember to do the `[please reassign]` trickstery to correctly assign it.